src/ui/views/ManualDownload.svelte

```diff
@@ -31,39 +110,81 @@
     />
   </div>
   <div class="col-6">
-    <h4 style="font-weight: bold;">Manual Download</h4>
+    <h4 style="font-weight: bold;">Manual Download Required</h4>
     <p>
-      The <b>{$manualDownloadGroup}</b> file
-      <b>{$manualDownloadFileData.name}</b> needs to be manually downloaded due to
+      The <strong>{$manualDownloadGroup}</strong> file
+      <strong>{$manualDownloadFileData.name}</strong> needs to be manually downloaded due to
       licensing restrictions. Sorry for the inconvenience!
     </p>
-    <p>
-      Please download the file <b>{$manualDownloadFileData.name}</b> from
-      <a href={$manualDownloadFileData.url}>{$manualDownloadFileData.url}</a>.
-    </p>
-    <p>
-      Once you have it, select it in the file picker and click the button below
-      to continue.
-    </p>
-    <div class="input-group" style="margin-bottom: 1em;">
-      <div class="custom-file">
-        <input
-          type="file"
-          bind:files={downloadedFile}
-          disabled={checkingFile}
-        />
+    
+    <div class="alert alert-info">
+      <strong>Step 1:</strong> Download the file from:<br/>
+      <a 
+        href={$manualDownloadFileData.url} 
+        target="_blank"
```

Comment:
What is this supposed to accomplish? External URLs are handled here https://github.com/ubports/ubports-installer/blob/509f56951d38624b0fddefffe9c278e55145326b/src/main.js#L90-L99 
This will actually be a security issue, as it will open the link in a new Electron window, with all the Node integration enabled.
Please remove this